sdk/keyvault/keyvault-admin/test/public/backupClient.spec.ts

```diff
@@ -2,77 +2,120 @@
-    assert.equal(backupResult, blobStorageUri);
+  describe("beginBackup", function() {
+    it("returns the correct backup result when successful", async function() {
+      const backupPoller = await client.beginBackup(
+        blobStorageUri,
+        blobSasToken,
+        testPollerProperties
+      );
+      const backupResult = await backupPoller.pollUntilDone();
+      assert.notExists(backupPoller.getOperationState().error);
+      assert.match(backupResult, new RegExp(blobStorageUri));
+    });
+
+    it("returns the correct backup result when fails to authenticate", async function() {
+      const backupPoller = await client.beginBackup(
+        blobStorageUri,
+        "invalid_sas_token",
+        testPollerProperties
+      );
+      const backupResult = await backupPoller.pollUntilDone();
+      assert.notExists(backupResult);
+      const operationState = backupPoller.getOperationState();
+      assert.isDefined(operationState.error);
+      assert.isNotEmpty(operationState.error?.message);
+    });
   });
 
-  it.skip("beginBackup, then beginRestore", async function() {
-    const blobStorageUri = env.BLOB_STORAGE_URI;
-    const sasToken = env.BLOB_STORAGE_SAS_TOKEN;
-    const backupPoller = await client.beginBackup(blobStorageUri, sasToken, testPollerProperties);
-    const backupURI = await backupPoller.pollUntilDone();
-    assert.ok(!!backupURI.match(blobStorageUri));
+  describe("beginRestore", function() {
+    it("full restore completes successfully", async function() {
+      const backupPoller = await client.beginBackup(
+        blobStorageUri,
+        blobSasToken,
+        testPollerProperties
+      );
+      const backupURI = await backupPoller.pollUntilDone();
+      const folderName = getFolderName(backupURI);
 
-    const folderName = getFolderName(backupURI);
-    const restorePoller = await client.beginRestore(
-      blobStorageUri,
-      sasToken,
-      folderName,
-      testPollerProperties
-    );
-    await restorePoller.pollUntilDone();
-    const operationState = restorePoller.getOperationState();
-    assert.equal(operationState.isCompleted, true);
-    assert.equal(operationState.error, undefined);
-  });
+      const restorePoller = await client.beginRestore(
+        blobStorageUri,
+        blobSasToken,
+        folderName,
+        testPollerProperties
+      );
+      await restorePoller.pollUntilDone();
+      const operationState = restorePoller.getOperationState();
+      assert.equal(operationState.isCompleted, true);
+      assert.notExists(operationState.error);
+      // Restore is eventually consistent so while we work
+      // through the retry operations adding a delay here allows
+      // tests to pass the 5s polling delay.
+      await delay(5000);
+    });
 
-  it.skip("beginBackup, then beginSelectiveRestore", async function() {
-    const keyName = "rsa-1";
+    it.skip("selectiveRestore completes successfully", async function() {
```

Comment:
Do you mind adding a comment on why this test is skipped? So that anybody can have more context when they see this.